src/nvim/tui/tui.c

```diff
@@ -208,10 +208,12 @@ static void try_resize(Event ev)
 static void sigwinch_cb(uv_signal_t *handle, int signum)
 {
   // Queue the event because resizing can result in recursive event_poll calls
+  /* FIXME(blueyed): this was changed to use a deferred event to fix #2322,
+   *                 without really knowing why it is necessary. */
```

Comment:
Ok, amended.
I wanted to keep the line length below 78 chars, but not use two prefixed line comments.